apps/design/backend/src/language_and_audio/translation_overrides.ts

```diff
@@ -0,0 +1,17 @@
+import { LanguageCode, NonEnglishLanguageCode } from '@votingworks/types';
+
+export type TranslationOverrides = Record<
+  NonEnglishLanguageCode,
+  { [englishText: string]: string | undefined }
+>;
+
+/**
+ * Cloud translations are sometimes incorrect and need to be globally overridden.
+ */
+export const GLOBAL_TRANSLATION_OVERRIDES: TranslationOverrides = {
```

Comment:
For now, we can update these global overrides by updating this object. As mentioned in the PR description, down the road, I imagine that we'll persist these overrides in the `Store` and expose some internal UI for updating them.